### DISCUSSION TITLE
[BUG]  could not find driver (SQL: select * from "countries" order by "name" asc)

### DISCUSSION THREAD

**Discussion by sayandeep-volition** [squirephp/squire#30] [Q&A]
**Describe the bug**
Installed the package then when I was going to use it in the blade that time I got this error.

**To reproduce**
`  @foreach (Squire\Models\Country::orderBy('name')->get() as $country)
                                            <option value="{{ $country->id }}">{{ $country->name }} {{ $country->flag }}</option>
                                        @endforeach`
**Expected behavior**
All country list

**Context**
- squirephp/countries-en: 2.1
- Laravel/framework:  8.40"
- Windows 10
- PHP 7.4

**Additional details**

Did I skip any steps?

**Comment by danharrin** [comment]
Hey, do you have the SQLite extension installed for PHP?

**Reply by nettsite** [reply]
I am having the same problem, and I do have SQLite installed (Composer wouldn't install your package without it):
```
SQLite 3.x driver for PDO	Wez Furlong
SQLite3	Scott MacVicar, Ilia Alshanetsky, Brad Dewar
```

database.php:
```
'sqlite' => [
            'driver' => 'sqlite',
            // 'url' => env('DATABASE_URL'),
            // 'database' => env('DB_DATABASE', database_path('database.sqlite')),
            'database' => database_path('database.sqlite'),
            'prefix' => '',
            'foreign_key_constraints' => env('DB_FOREIGN_KEYS', true),
        ],
```

Php 8.3, Laravel 10, Filament 3.
